src/extensions/apollo/index.ts

```diff
@@ -0,0 +1,21 @@
+import { jsonToGraphQLQuery } from '../../index';
+
+export function addApolloTargetToGqlQueryString(query: string, target: string): string {
+    for (let i = query.length; i--; i === 0) {
+        const thisChar = query[i];
+        if ((thisChar !== ' ') && (thisChar !== '}')) {
+            return query.slice(0, i + 1) + ' @' + target + query.slice(i + 1 , query.length);
+        }
+    }
+}
+
+export function objToApolloQuery(object: object, target: string): string {
+    const gqlQuery = jsonToGraphQLQuery(object);
+    const querySansTypenames = gqlQuery.replace(/__typename /g, '');
```

Comment:
Another way to get rid of all __typename keys, would be to add it to the list of objects keys to ignore, here:

https://github.com/dupski/json-to-graphql-query/blob/master/src/jsonToGraphQLQuery.ts#L51

(all those conditions on that line should probably be added to an array - it feels like that would make a good library 'option' too - "keysToRemove", in case there are other keys that people want stripped)

Reply:
Regarding __typename, I concur with both ideas:
- Short term: Adding "__typename" to list of keys to ignore
- Long term: Also think a keysToRemove option.

Regarding the merge conflict, np, I'll fetch and resolve it.